internal/pkg/deploy/cloudformation/app.go

```diff
@@ -279,7 +284,7 @@ func (cf CloudFormation) addWorkloadToApp(app *config.Application, wlName string
 		Accounts: previouslyDeployedConfig.Accounts,
 		App:      appConfig.Name,
 	}
-	if err := cf.deployAppConfig(appConfig, &newDeploymentConfig); err != nil {
+	if err := cf.deployAppConfig(appConfig, &newDeploymentConfig, shouldAddNewWl); err != nil {
```

Comment:
isn't this always true?

(i'm not sure i'd even want switch it to just `true`, but just <img src="https://emojis.slackmojis.com/emojis/images/1643509571/47561/thinkspin.gif" data-canonical-src="https://emojis.slackmojis.com/emojis/images/1643509571/47561/thinkspin.gif" width="20" height="20" />)

Reply:
yeah i like it the way you have it 😊